Dockerfile

```diff
@@ -2,8 +2,19 @@ FROM alpine:latest as alpine
 
 RUN apk add -U --no-cache ca-certificates
 
+# Build a statically linked git binary
+ENV GIT_VERSION "v2.30.0"
+WORKDIR /tmp/git
+RUN apk add -U --no-cache git alpine-sdk autoconf automake zlib-dev zlib-static zlib
+RUN \
+  git clone --depth=1 git://github.com/git/git -b $GIT_VERSION /tmp/git && \
+  make configure && \
+  ./configure prefix=/tmp CFLAGS="${CFLAGS} -static" && \
+  make install NO_TCLTK="YesPlease"
+
 FROM scratch
 WORKDIR /workdir
-COPY pulse-event-cli /
+COPY pulse-event-cli /bin/
 COPY --from=alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-ENTRYPOINT ["/pulse-event-cli"]
+COPY --from=alpine /tmp/bin/git /bin/git
+ENTRYPOINT ["pulse-event-cli"]
```

Comment:
shouldn't it be /bin/pulse-event-cli ?
Did you try to build and run the docker?

Reply:
Since we place it in /bin it is in the path.
Since git needs to be in the PATH to be found I tough for consistency I would put both.

Reply:
ahh, you are right 👍